prow/git/v2/client_factory.go

```diff
@@ -219,19 +210,26 @@ func (c *clientFactory) bootstrapClients(org, repo, dir string) (cacher, cloner,
 	if err != nil {
 		return nil, nil, nil, err
 	}
+	var remote RemoteResolverFactory
+	remote = c.remotes
```

Comment:
Nit: This feels odd, assigning the value of a plural noun `remotes` to a singular noun `remote`. One or the other should probably be named differently.

Reply:
yeah I think `remote` makes more sense, updated